test/xspec.bats

```diff
@@ -190,3 +190,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "${lines[3]}" =~ "Testing with" ]]
 }
+
+@test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file:///home/travis/build/xspec/xspec/ -oresult=/home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc && grep 'charset=UTF-8' /home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html"
```

Comment:
We have to avoid hardcoding the path.
When the repository is forked, the path is different from `xspec/xspec`. The test always fails there as follows.

```shell
not ok 20 executing the Saxon XProc harness generates a report with UTF-8 encoding
# (in test file xspec.bats, line 198)
#   `[ "$status" -eq 0 ]' failed
# 1
# ERROR: err:XD0011:XProc error err:XD0011 ERROR: cause: I/O error reported by XML parser processing file:/home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc: /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc (No such file or directory) ERROR: It is a dynamic error if the resource referenced by a p:document element does not exist, cannot be accessed, or is not a well-formed XML document. ERROR: Underlying exception: I/O error reported by XML parser processing file:/home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc: /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc (No such file or directory)
```
Also [this article](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables) denotes `/home/travis` as `(do not depend on this value)`.